From 29955abce07526d01f3298da9d2f15bd9eb5e6f4 Mon Sep 17 00:00:00 2001
From: Andreas Sturmlechner <andreas.sturmlechner@gmail.com>
Date: Sun, 18 Nov 2018 18:07:32 +0100
Subject: [PATCH] Switch to tidy-html5 and use tidybuffio.h instead of buffio.h

Summary: Header was renamed a few years ago.

Test Plan: Builds and validates fine.

Reviewers: #konqueror

Differential Revision: https://phabricator.kde.org/D16982
---
 plugins/validators/CMakeLists.txt     | 2 +-
 plugins/validators/tidy_validator.cpp | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/plugins/validators/CMakeLists.txt b/plugins/validators/CMakeLists.txt
index cf3b70694..7f6060e9c 100644
--- a/plugins/validators/CMakeLists.txt
+++ b/plugins/validators/CMakeLists.txt
@@ -4,7 +4,7 @@ set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${CMAKE_CURRENT_SOURCE_DIR})
 find_package(LibTidy)
 find_package(KF5 REQUIRED KHtml WidgetsAddons IconThemes)
 set_package_properties(LibTidy PROPERTIES DESCRIPTION "HTML Tidy"
-                       URL "http://tidy.sourceforge.net"
+                       URL "http://www.html-tidy.org/"
                        TYPE OPTIONAL
                        PURPOSE "Gives the Konqueror validators plugin the ability to validate HTML"
                       )
diff --git a/plugins/validators/tidy_validator.cpp b/plugins/validators/tidy_validator.cpp
index 3247199f9..ac22429d0 100644
--- a/plugins/validators/tidy_validator.cpp
+++ b/plugins/validators/tidy_validator.cpp
@@ -25,7 +25,7 @@
 
 #include <kdebug.h>
 
-#include <buffio.h>
+#include <tidybuffio.h>
 #include <tidy.h>
 
 #include <config-konq-validator.h>
-- 
2.19.1

